sidekick_core/lib/src/commands/plugins/create_templates/shared_code.dart

```diff
@@ -64,22 +68,29 @@ Future<void> main() async {
   }
   pubGet(package);
   
-  final commandFile = package.root.file('lib/src/${pluginName.snakeCase}.dart');
-  commandFile.writeAsStringSync("""
-$exampleCommand
-""");
+  // Platform.script.path -> <plugin>/tool/install.dart
+  final pluginDir = File(Platform.script.path).parent.parent;
+
+  final cliCommandFile =
+      package.root.file('lib/src/${pluginName.snakeCase}_command.dart');
```

Comment:
I do not like that the command file is located in `lib/src/`. It suggests that the command is in scope of the plugin. Plugin authors might accidentally import code, from the plugin scope that isn't available in the generated plugin.

I'd prefer some indicator, that the file is not part of the plugin itself. Like a `.template.dart` file extension of placing it in `<package_root>/template/`.

Reply:
That's very reasonable 👍 Just to clarify what I implemented now:

When generating a plugin, the command file template is placed in `<plugin_root>/template/<plugin_name>_command.template.dart`

When installing that plugin to a cli, the command file is still placed in `<cli_root>/lib/src/<plugin_name>_command.dart`